src/cljs/athens/views/blocks/core.cljs

```diff
@@ -362,7 +362,18 @@
                 reactions              (and reactions-enabled?
                                             (block-reaction/props->reactions properties))
                 menu                   (r/as-element
-                                         [:<>
+                                         [:> MenuGroup {:className ".anchor"}
```

Comment:
Copying multiple block refs or text from multiple selected blocks only copies from one. Class name might need to be `anchor`, not `.anchor`.